include/af/graphics.h

```diff
@@ -154,6 +164,28 @@ class AFAPI Window {
         void hist(const array& X, const double minval, const double maxval, const char* const title=NULL);
 
         /**
+           Renders the input arrays as a 3D surface plot to the window
+
+           \param[in] S is an \ref array with the z-axis data points
+           \param[in] title parameter is used when this function is called in grid mode
+
+           \note \p S should be s 2D array
+         */
+void surface(const array& S, const char* const title);
+
+        /**
+           Renders the input arrays as a 3D surface plot to the window
+
+           \param[in] X is an \ref array with the x-axis data points
```

Comment:
Variable names for first two parameters in `surface` are not same as function signature, this will throw errors in doxygen.